spec/features/editing_a_manual_section_spec.rb

```diff
@@ -0,0 +1,62 @@
+require 'spec_helper'
+
+RSpec.feature "editing a manual section" do
+  context 'as a GDS editor' do
+    let(:manual) { Payloads.manual_content_item }
+    let(:manual_links) { Payloads.manual_links }
+    let(:sections) { Payloads.section_content_items }
+    let(:sections_links) { Payloads.section_links }
+
+    let(:manual_content_id) { manual['content_id'] }
+
+    let(:section_content_id) { sections.first['content_id'] }
+
+    before do
+      log_in_as_editor(:gds_editor)
+
+      publishing_api_has_item(manual)
+      publishing_api_has_links(manual_links)
+
+      sections.each { |section| publishing_api_has_item(section) }
+      sections_links.each { |section_links| publishing_api_has_links(section_links) }
+
+      stub_publishing_api_put_content(section_content_id, {})
+      stub_publishing_api_patch_links(section_content_id, {})
+    end
+
+    scenario 'editing a section of a manual' do
+      visit manual_path(manual_content_id)
+      click_link 'First section'
+      click_link 'Edit section'
+
+      expect(page.status_code).to eq(200)
+
+      expect(page).to have_field('Title')
```

Comment:
You could probably remove these `expect(page).to have_field(x)` lines, as the `fill_in` on the next line will also run a `find` on the element and fail if it can't find it.